wagtail/contrib/redirects/forms.py

```diff
@@ -42,3 +44,51 @@ def clean(self):
     class Meta:
         model = Redirect
         fields = ('old_path', 'site', 'is_permanent', 'redirect_page', 'redirect_link')
+
+
+class ImportForm(forms.Form):
+    import_file = forms.FileField(label=_("File to import"))
+    input_format = forms.ChoiceField(label=_("Format"), choices=[],)
```

Comment:
@lb- No problem at all, I actually do this in the cli, so this would make it more consistent. I'm considering if we should keep the `input_format` field, but make it optional so we can support cases where the user wants to override the file format. Any thoughts on this?

Reply:
I think less fields is better, if the user wants to upload a CSV with the extension .tsv or a .txt - I think that is a really small edge case and most average users would not even understand that extensions can be anything.

Reply:
One additional thought - It would be good to restrict the file extensions in the field upload widget to what is supported only. 

Similar to this - https://stackoverflow.com/a/43737771/8070948

Reply:
Awesome, then we drop the field and use a customized upload widget instead.